samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -247,7 +247,7 @@ select_the_feedback=Select the Feedback Components to be displayed to the studen
 selected=Selected
 selection_level_feed=Selection-level Feedback
 time_expired1=Time Expired!
-time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted.
+time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted in the next few minutes.
```

Comment:
@fgeraci ya I understand that part, sorry maybe I wasn't very clear. I'm just curious when the automatic submission occurs, given the following String you're changing:

`The time allowed for this assessment has expired. Your answers have been automatically submitted in the next few minutes.`

Do you know exactly when the automatic submission occurs? I'm just thinking this String could be more specific, so as to be less ambiguous to the user taking the assessment.